src/views/modules/bar.php

```diff
@@ -26,6 +26,8 @@
 <?php do_action( 'tribe_events_bar_before_template' ) ?>
 <div id="tribe-events-bar">
 
+	<h2 class="tribe-events-visuallyhidden"><?php printf( esc_html__( '%s Search and Views Navigation', 'the-events-calendar' ), tribe_get_event_label_plural() ); ?></h2>
```

Comment:
Could we use `tribe-events-visually-hidden`

Reply:
@bordoni `tribe-events-visuallyhidden` is an existing CSS class which I used for this context. Should I add an additional class name to that declaration with the new name (`tribe-events-visually-hidden`)? Or just keep this as-is?

Reply:
Ohh sorry, didn't realize that. Let's keep it as is.